src/runnable.rs

```diff
@@ -30,6 +39,114 @@ impl<M: Default> Default for Builder<M> {
     }
 }
 
+/// Extra scheduling information that can be acquired by the scheduling
```

Comment:
"Acquired" is a weird word here, maybe try "passed" instead?

Reply:
Changed the phrase to "passed to".